consensus/src/experimental/tests/commit_phase_tests.rs

```diff
@@ -512,6 +522,9 @@ mod commit_phase_function_tests {
                 Err(Error::InconsistentBlockInfo(_, _))
             ));
 
+            /*
+            // Now the verification is handled inside epoch_manager.
+            // Commit phase does not check the signatures aymore.
```

Comment:
just remove unused section instead of commented out?

Reply:
resolved